snap/snapcraft.yaml

```diff
@@ -25,3 +27,38 @@ parts:
     plugin: dump
     organize:
       '*': usr/share/doc/boot-firmware/
+
+  # Note: The device tree for EVK boards are pre-compiled in the kernel, so we
+  #       could just use it. But for ODM vendor, you need to prepare your own
+  #       device-tree following by ubuntu document.
```

Comment:
Hi @taiten 
As far as I know, you should be the person to provide the document to customers. Is it correct?
Hi @Kxuan 
If it's correct, it seems to me that we need to add something like "Please contact Canonical account field engineer to get the document for building device tree".